mfthomps/CyberCIEGE-port-to-Unity3d
# Issue/35 computer configuration settings

Closes #35 

This list two different categories of "Policies" in the Computer configuration screen and in the Zone screen. In the game, they are referred to as "Procedural Settings" and "Configuration Settings" -- I'm not sure what the difference is.

In the Zone Configuration screen, these are supposed to setup the default policies for all new equipment purchased (in that Zone?).

To verify:
* right-click -> Configure on any hardware in the scenario
* The pop up screen should have two columns of policies that mimic the two columns in the original UI.
* Compare the original UI with the Unity version.
* Right-click -> Zone -> Entire Office
* This should show a similar two-column list of Policies and is meant to enable/disable the policies for all new purchased equipment. Note: this doesn't seem to work in the Unity version, but does in the original UI.

Computer Configuration
![image](https://user-images.githubusercontent.com/7308321/104484677-6bef0980-557e-11eb-8991-1fa9d0f981b3.png)

Zone Configuration
![image](https://user-images.githubusercontent.com/7308321/104484972-c38d7500-557e-11eb-9940-e6e951a15079.png)

Original
![image](https://user-images.githubusercontent.com/7308321/104484777-8628e780-557e-11eb-9b50-c1db9195f32d.png)
